ui/components/Shared/SharedAssetInput.tsx

```diff
@@ -20,6 +20,32 @@ import SharedAssetItem, {
 } from "./SharedAssetItem"
 import SharedAssetIcon from "./SharedAssetIcon"
 
+// List of symbols we want to display first.  Lower array index === higher priority.
+// For now we just prioritize somewhat popular assets that we are able to load an icon for.
+const SYMBOL_PRIORITY_LIST = [
+  "UST",
```

Comment:
Based on the top tokens on Uniswap,
```suggestion
  "DAI ",
  "USDC",
  "USDT",
  "WBTC",
  "UST",
```
How about we include these?

Reply:
I am happy to and I looked at the top tokens on uniswap as well - but the only issue is that none of those tokens except `UST` have icons in tally (yet) so it looks pretty bad when opened with them at the top.

Reply:
Ah ah good point